homeassistant/components/lamarzocco/__init__.py

```diff
@@ -99,18 +105,26 @@ def bluetooth_configured() -> bool:
                 address_or_ble_device=entry.data[CONF_MAC],
             )
 
-    coordinator = LaMarzoccoUpdateCoordinator(
-        hass=hass,
-        entry=entry,
-        local_client=local_client,
+    device = LaMarzoccoMachine(
+        model=entry.data[CONF_MODEL],
+        serial_number=entry.unique_id,
+        name=entry.data[CONF_NAME],
         cloud_client=cloud_client,
+        local_client=local_client,
         bluetooth_client=bluetooth_client,
     )
 
-    await coordinator.async_config_entry_first_refresh()
-    entry.runtime_data = coordinator
+    coordinators = (
+        LaMarzoccoConfigUpdateCoordinator(hass, entry, device, local_client),
+        LaMarzoccoFirmwareUpdateCoordinator(hass, entry, device),
+        LaMarzoccoStatisticsUpdateCoordinator(hass, entry, device),
+    )
+    for coordinator in coordinators:
+        await coordinator.async_config_entry_first_refresh()
+
+    entry.runtime_data = coordinators[0]
```

Comment:
What is the point of the other coordinators if we don't keep them?

Reply:
I spoke to joost and we thought they should run in the background nonetheless, or do they not?

Reply:
There is nothing referencing these objects, meaning they will be cleaned up by Python. Additionally, there is nothing listening to these coordinators; coordinators without listeners won't poll.